src/systemtest/java/reposense/ConfigSystemTest.java

```diff
@@ -65,7 +66,7 @@ private void generateReport(String inputDates) throws IOException, URISyntaxExce
         Path configFolder = Paths.get(getClass().getClassLoader().getResource("repo-config.csv").toURI()).getParent();
 
         String formats = String.join(" ", TESTING_FILE_FORMATS);
-        String input = String.format("-config %s -formats %s ", configFolder, formats) + inputDates;
+        String input = new InputBuilder().addConfig(configFolder).addFormats(formats).add(inputDates).build();
```

Comment:
Let's have some line breaks here after adding each option and arugment, to make it more readable:
```
new InputBuilder().addConfig(configFolder)
        .addFormats(formats)
        .add(inputDates)
        .build();
```